README.md

```diff
@@ -71,16 +71,26 @@ The HTTP response, in this case would be:
     "ongoing": true,
     "product_id": 1,
     "start_date": "01-01-2022 10:10:10 ",
-    "type": "Value",
+    "type": "VALUE",
```

Comment:
I believe "Value" is a hardcoded string which indicates "Value" in a Enum (line 49 in service/models.py) and the type in json object will be deserialized at line 155. I don't think "Value" is equivalent to "VALUE".

Reply:
I just realized the latest commit has changed all constants in upper-case. Never mind.

Reply:
Yes, you are right. I thought I changed it. For some reason, it worked the other day, but I tried the same thing today, and as you said, 'Value' was not recognized.